tests/rptest/tests/cluster_bootstrap_test.py

```diff
@@ -57,18 +66,26 @@ def test_three_node_bootstrap(self, num_seeds, auto_assign_node_ids):
             assert expected_node_id == node_id, f"Expected {expected_node_id} but got {node_id}"
 
 
-class ClusterBootstrapUpgradeTest(RedpandaTest):
+class ClusterUpgrade(RedpandaTest):
```

Comment:
nit: please include "Bootstrap" in the name, as this is testing upgrade specifically focused on cluster bootstrap configurations. Otherwise it's very ambiguous what this is testing.

Reply:
Bootstrap is a part of module name `cluster_bootstrap_test.py` which is a part of the full name of the test, that's why I've called it just "upgrade". But ok, will do.

Reply:
done